File: python/exercício_87.py
palavra1= str(input("Digite algo: "))
palavra2= str(input("\nDigite outra palavra: "))

lower1= palavra1.lower()
lower2 = palavra2.lower()


if lower1 == lower2:
    print(f'\nA palavra {palavra1} e a {palavra2} são iguais!\n ')
else: 
    print(f'\nA palavra {palavra1} e a  {palavra2} são diferentes\n')

File: java/exercício_87.java
import java.util.Scanner;

public class exercício_87 {
    public static void main(String[] args) {
        try (Scanner texto = new Scanner(System.in)) {
            System.out.print("Digite a sua primeira palavra: ");
            String palavra = texto.nextLine();
            System.out.print("Digite a sua segunga palavra: ");
            String palavra1 = texto.nextLine();

            if (palavra.equalsIgnoreCase(palavra1)) {
                System.out.print("os textos " + palavra + " e " + palavra1 + " são iguais");
            } else {
                System.out.print("os textos " + palavra + " e " + palavra1 + " são diferentes");
            }
        }

    }

}

                                        
